packages/test/test-end-to-end-tests/src/test/.mocharc.js

```diff
@@ -12,6 +12,7 @@ const testPackagesDir = `${packageDir}/..`;
 const requiredModules = [
     `${testPackagesDir}/mocha-test-setup`, // General mocha setup e.g. suppresses console.log
     `${testPackagesDir}/test-drivers`, // Inject implementation of getFluidTestDriver, configured via FLUID_TEST_DRIVER
+    `./dist/test/testApi`,
```

Comment:
can we just put this in another package? i know you will refactor is a follow up and this is just dynamic loading, not really the injection part, but we will need a seperate pacakge for injection, and it will help keep the tests decoupled

Reply:
Yep... that's the plan later.  I haven't examine where to put yet, a new package or test-utils.   Again, I don't want to complicate this change with additional stuff.